src/components/Home/VoteParticipation.tsx

```diff
@@ -32,13 +28,12 @@ export default function VoteParticipation() {
   useLoadSectionRefAndId(ref, id);
 
   useEffect(() => {
-    if (inView && !stakeData) {
+    if (!stakeData) {
```

Comment:
only loading these when in view also contributed to the scrolling problem — they need to be loaded immediately so that we can know their heights and include that in the section coordinates calculation